### PR TITLE
Adding script to download specific indicators from world bank WDI dataset

### PR DESCRIPTION
**Pull request by cbari123**
This script will download data from world bank for a explicit set of indicators and write a cleaned csv to a given output path

### CODE REVIEW

**Review by pradh — COMMENTED** [review]
Maybe drop a short README?

**Comment by cbari123** [comment]
> Maybe drop a short README?

Added a very simple README

File: scripts/world_bank/wdi/wdi_download_indicators.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
+"""Download World Bank WDI via API and write cleaned csv to out_path."""
```
Comment (pradh) [inline scripts/world_bank/wdi/wdi_download_indicators.py:1]: Maybe remove this duplicate file?

**Review by pradh — APPROVED** [review]
Thanks Chase!

**Pull request merged**